x-pack/plugins/security_solution/public/notes/telemetry/types.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { RootSchema } from '@kbn/ebt';
```

Comment:
why is it pulling from `@kbn/ebt` instead of `@kbn/core/public`?

Reply:
not sure, I let Webstorm do the import on its own and didn't pay attention... Reverted!

Reply:
removed in my last commit